core/src/main/scala/ai/lum/odinson/Mention.scala

```diff
@@ -39,4 +39,15 @@ case class Mention(
       }
   }
 
+  def copy(mentionFactory: MentionFactory /*= new DefaultMentionFactory*/,
```

Comment:
@kwalcock do I understand correctly that this would allow for making a new "view" of the extraction with a different factory?  If so -- seems like this may be my "entry point" for having a custom mentionfactory/mention in eidosdottir

Reply:
Probably not, if I understand the question correctly.  This line in ExtractorEngine is where I intended  customization to be incorporated.  The engine gets a single factory when it is constructed which is hopefully good for the engine's entire lifespan.  It might be slipped in elsewhere but this was the main point of attack.

def fromDirectory(config: Config, indexDir: Directory, mentionFactory: MentionFactory = defaultMentionFactory): ExtractorEngine = {

Reply:
But that copy method could be used for a one-off.